websites/R/RTLplay/presence.ts

```diff
@@ -0,0 +1,423 @@
+					  ).textContent;
+			}
+
+			presenceData.smallImageKey = Assets.Reading;
+			presenceData.smallImageText = (await strings).browsing;
+
+			if (time) presenceData.startTimestamp = browsingTimestamp;
+
+			if (buttons) {
+				presenceData.buttons = [
+					{
+						label: (await strings).buttonViewPage,
+						url: href,
+					},
+				];
+			}
+
+			break;
+		}
+
+		/* Video player 
+	Lecteur vidéo
+	(ex: https://www.rtlplay.be/the-power-p_25630/episode-01-c_13062282) */
+		case document.querySelector("div.sc-18trp4n-3.bVsrtw > h1") !== null: {
+			presenceData.details = privacy
+				? (await strings).watchingShow
+				: `${(await strings).watching} ${
+						document.querySelector("div.sc-18trp4n-3.bVsrtw > h1").textContent
+				  }`; // Content of first line
+			// If 1st line === 2nd line then it's a Movie, 1st line !== 2nd line then it's a Tv Show
+			if (
+				document
+					.querySelector("div.sc-18trp4n-3.bVsrtw > h1")
+					.textContent.toLowerCase() !==
+				document
+					.querySelector("div.sc-3kw2pp-0.jqFqPu.sc-18trp4n-6.cQaAsL > h2")
+					.textContent.toLocaleLowerCase()
+			) {
+				presenceData.state = privacy
+					? ""
+					: document.querySelector(
+							"div.sc-3kw2pp-0.jqFqPu.sc-18trp4n-6.cQaAsL > h2"
+					  ).textContent; // Content of second line
+			}
+
+			presenceData.largeImageKey = Assets.Logo;
+			if (
+				document
+					.querySelector("#player-control-playpause")
+					.getAttribute("aria-label") === "Lecture"
+			) {
+				presenceData.smallImageKey = Assets.Pause;
+				presenceData.smallImageText = (await strings).pause;
+			} else {
+				presenceData.smallImageKey = Assets.Play;
+				presenceData.smallImageText = (await strings).play;
+			}
+
+			if (time) {
+				if (
+					document.querySelector('[type="duration"]').textContent.includes("-")
+				) {
+					presenceData.endTimestamp = presence.getTimestamps(
+						0,
+						presence.timestampFromFormat(
+							document
+								.querySelector('[type="duration"]')
+								.textContent.replace("-", "")
+						)
+					)[1];
+				} else {
+					presenceData.endTimestamp = presence.getTimestamps(
+						presence.timestampFromFormat(
+							document.querySelector('[type="currentTime"]').textContent
+						),
+						presence.timestampFromFormat(
+							document.querySelector('[type="duration"]').textContent
+						)
+					)[1];
+				}
+			}
+
+			if (buttons) {
+				presenceData.buttons = [
+					{
+						label: (await strings).buttonWatchVideo,
+						url: href, // We are not redirecting directly to the raw video stream, it's only the player page
+					},
+				];
+			}
+
+			break;
+		}
+
+		// In case we need a default
+		default: {
+			break;
+		}
+	}
+
+	if ((presenceData.startTimestamp || presenceData.endTimestamp) && !time) {
+		delete presenceData.startTimestamp;
+		delete presenceData.endTimestamp;
+	}
+	if (presenceData.details === "") delete presenceData.details;
+	if (presenceData.state === "") delete presenceData.state;
```

Comment:
```suggestion
```

Reply:
Does Premid automatically do that so those lines are not necessary ?
I added those lines to respect presence.ts rules
https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets (9th dot).